apps/backend/src/modules/project-groups/project-group.repository.ts

```diff
@@ -129,14 +129,16 @@ class ProjectGroupRepository implements Repository {
 			.withGraphJoined("[projects, users, permissions]")
 			.where("users.id", userId);
 
-		return results.map(({ projects, ...projectGroup }) => {
-			const [project] = projects;
+		return results
+			.filter(({ projects }) => projects.length)
```

Comment:
This check for cases when user deletes project, which has some project groups. After such a deletion in the DB project groups are remained in `project_groups` table, but they relation with `projects` are lost (corresponding record in `projects_to_project_groups` are deleted). So without this `.filter`, `project` in `const [project] = projects;` expression is undefined which leads to `Reading property 'id' of undefined` error.

It would be better to remove this filter after we fix mentioned issue.